build.gradle

```diff
@@ -97,9 +103,7 @@ dependencies {
     testImplementation 'com.intellij.remoterobot:remote-robot:' + remoteRobotVersion
     testImplementation 'com.intellij.remoterobot:remote-fixtures:' + remoteRobotVersion
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.9.2'
-    testImplementation('com.jetbrains.intellij.maven:maven-test-framework:241.15989.150') {
```

Comment:
why is this being removed? This will eventually be updated later, but should be left alone for now.

Reply:
Removed this dependency as jetbrains have introduced new TEST Framework which will automatically include this dependency based on intellij version in our project. This was suggested by jetbrains team here
https://jetbrains-platform.slack.com/archives/C5U8BM1MK/p1730297530842639?thread_ts=1730105939.390899&cid=C5U8BM1MK

Reply:
So we have confirmed that we can run the unit tests with this dependency removed?

Reply:
Yes, @TrevCraw,

Instead of adding maven-test-framework directly, IntelliJ 2024.2 introduced a way to add this line: testFramework TestFrameworkType.Plugin.Maven.INSTANCE within the intellijPlatform setup under the dependencies tag. This approach ensures that the dependency is downloaded in our project, and the version is determined by the IntelliJ platform version specified.

However, even with this adjustment, unit tests do not work in IntelliJ 2024.2. That said, making this change has been verified to fully support unit tests in IntelliJ 2024.1.

These are the dependencies used in the project, and when removing the direct dependency and instead adding the framework as suggested by JetBrains, maven-test-framework is downloaded correctly.
<img width="1641" alt="Screenshot 2024-11-15 at 11 21 57 AM" src="https://github.com/user-attachments/assets/a2a012d3-a5ec-486c-9548-aebed9a11dca">

Thanks,